.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/reviews
+    docker:
+      - image: circleci/node:latest
+      - image: 'circleci/mongo:4.4.1'
+    steps:
+      - checkout
+      - restore_cache:
+          key: npm-cache-v1-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install Dependencies
+          command: npm ci
+      - save_cache:
+          key: npm-cache-v1-{{ checksum "package-lock.json" }}
+          paths:
+            - /home/circleci/.npm
+      - run:
+          name: Start Server and Run Tests
+          command: npm start & npm test
```

Comment:
Nicely done! Maybe we can talk about this some time!

Reply:
Sounds great, anytime!